asllib/doc/Types.tex

```diff
@@ -190,15 +200,15 @@ \subsubsection{Prose}
       \item $\tty$ is an integer type that is not well-constrained;
       \item $\newty$ is the unconstrained integer type.
     \end{itemize}
-  \item All of the following apply (\textsc{well\_constrained\_empty}):
+  \item All of the following apply (\textsc{pending\_constrained}):
     \begin{itemize}
-      \item $\tty$ is the well-constrained integer type with no constraints;
+      \item $\tty$ is a pending constrained integer type;
```

Comment:
Ideally, use use a macro `\pengingconstrainedintegertypeterm` as a  "glossary" macro (search for 'glossary' in ASLMacros.tex) and use it here.